webviews/createPullRequestView/index.css

```diff
@@ -84,4 +84,8 @@ body {
 
 .selector-group .wrapper {
 	padding-right: 5px;
-}
\ No newline at end of file
+}
+.select-style {
+	background-color: #3d3c3d;
```

Comment:
Instead of hardcoding a value, we should use one of the VS Code theme colors so the solution works with all themes. The same one we use for the select options should be ok:

https://github.com/yash621/vscode-pull-request-github/blob/61b93b263b36f1b7d8c93096d439e3853aca7428/webviews/activityBarView/index.css#L115-L116